governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021
+summary: MIP40c3-SP39 extends the budget for the Real-World Finance Core Unit defined in MIP40c3-SP12 in 2022.
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp39-modify-core-unit-budget-rwf-001/10543
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-08T16:00:00 
+end_date: 2021-11-22T16:00:00
+---
+# Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, November 8 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP39 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+With [MIP40c3-SP12](https://forum.makerdao.com/t/mip40c3-sp12-modify-core-unit-budget-rwf-001/7985) the Real-World Finance team was able to expand the team, sustain legal costs related to RWA, onboard more RWA and develop FP&A. This new budget extends the previous one in 2022.
```

Comment:
I'd probably link to [the MIPs Portal](https://mips.makerdao.com/mips/details/MIP40c3SP12) here. Also, formatting: This should be cursive

Reply:
Fixed. I just copied over the link that was used in the MIPs Portal doc but agree this is better

Reply:
Yes. Looks like I missed that one on the proposal :[